diff -up dpdk-2.0.0/config/common_linuxapp.orig dpdk-2.0.0/config/common_linuxapp
--- dpdk-2.0.0/config/common_linuxapp.orig	2015-04-03 10:13:45.000000000 -0400
+++ dpdk-2.0.0/config/common_linuxapp	2015-04-06 10:52:14.073329962 -0400
@@ -52,7 +52,7 @@ CONFIG_RTE_EXEC_ENV_LINUXAPP=y
 ## Note: if your compiler does not support the relevant -march options,
 ## it will be compiled with whatever latest processor the compiler supports!
 ##
-#CONFIG_RTE_MACHINE="native"
+CONFIG_RTE_MACHINE="default"
 #
 ##
 ## define the architecture we compile for.
@@ -98,7 +98,7 @@ CONFIG_RTE_LOG_HISTORY=256
 CONFIG_RTE_LIBEAL_USE_HPET=n
 CONFIG_RTE_EAL_ALLOW_INV_SOCKET_ID=n
 CONFIG_RTE_EAL_ALWAYS_PANIC_ON_ERROR=n
-CONFIG_RTE_EAL_IGB_UIO=y
+CONFIG_RTE_EAL_IGB_UIO=n
 CONFIG_RTE_EAL_VFIO=y
 
 #
@@ -147,8 +147,8 @@ CONFIG_RTE_NIC_BYPASS=n
 #
 # Compile burst-oriented IGB & EM PMD drivers
 #
-CONFIG_RTE_LIBRTE_EM_PMD=y
-CONFIG_RTE_LIBRTE_IGB_PMD=y
+CONFIG_RTE_LIBRTE_EM_PMD=n
+CONFIG_RTE_LIBRTE_IGB_PMD=n
 CONFIG_RTE_LIBRTE_E1000_DEBUG_INIT=n
 CONFIG_RTE_LIBRTE_E1000_DEBUG_RX=n
 CONFIG_RTE_LIBRTE_E1000_DEBUG_TX=n
@@ -159,7 +159,7 @@ CONFIG_RTE_LIBRTE_E1000_PF_DISABLE_STRIP
 #
 # Compile burst-oriented IXGBE PMD driver
 #
-CONFIG_RTE_LIBRTE_IXGBE_PMD=y
+CONFIG_RTE_LIBRTE_IXGBE_PMD=n
 CONFIG_RTE_LIBRTE_IXGBE_DEBUG_INIT=n
 CONFIG_RTE_LIBRTE_IXGBE_DEBUG_RX=n
 CONFIG_RTE_LIBRTE_IXGBE_DEBUG_TX=n
@@ -173,7 +173,7 @@ CONFIG_RTE_IXGBE_RX_OLFLAGS_ENABLE=y
 #
 # Compile burst-oriented I40E PMD driver
 #
-CONFIG_RTE_LIBRTE_I40E_PMD=y
+CONFIG_RTE_LIBRTE_I40E_PMD=n
 CONFIG_RTE_LIBRTE_I40E_DEBUG_INIT=n
 CONFIG_RTE_LIBRTE_I40E_DEBUG_RX=n
 CONFIG_RTE_LIBRTE_I40E_DEBUG_TX=n
@@ -242,7 +242,7 @@ CONFIG_RTE_PMD_RING_MAX_TX_RINGS=16
 #
 # Compile software PMD backed by PCAP files
 #
-CONFIG_RTE_LIBRTE_PMD_PCAP=n
+CONFIG_RTE_LIBRTE_PMD_PCAP=y
 
 #
 # Compile link bonding PMD library
@@ -338,7 +338,7 @@ CONFIG_RTE_LIBRTE_LPM_DEBUG=n
 #
 # Compile librte_acl
 #
-CONFIG_RTE_LIBRTE_ACL=y
+CONFIG_RTE_LIBRTE_ACL=n
 CONFIG_RTE_LIBRTE_ACL_DEBUG=n
 
 #
@@ -403,8 +403,8 @@ CONFIG_RTE_LIBRTE_PIPELINE=y
 #
 # Compile librte_kni
 #
-CONFIG_RTE_LIBRTE_KNI=y
-CONFIG_RTE_KNI_PREEMPT_DEFAULT=y
+CONFIG_RTE_LIBRTE_KNI=n
+CONFIG_RTE_KNI_PREEMPT_DEFAULT=n
 CONFIG_RTE_KNI_KO_DEBUG=n
 CONFIG_RTE_KNI_VHOST=n
 CONFIG_RTE_KNI_VHOST_MAX_CACHE_SIZE=1024
diff -up dpdk-2.0.0/config/defconfig_x86_64-native-linuxapp-gcc.orig dpdk-2.0.0/config/defconfig_x86_64-native-linuxapp-gcc
--- dpdk-2.0.0/config/defconfig_x86_64-native-linuxapp-gcc.orig	2015-04-03 10:13:45.000000000 -0400
+++ dpdk-2.0.0/config/defconfig_x86_64-native-linuxapp-gcc	2015-04-06 10:53:01.495666675 -0400
@@ -31,8 +31,10 @@
 #
 
 #include "common_linuxapp"
-
-CONFIG_RTE_MACHINE="native"
+# Note: if your compiler does not support the relevant -march options,
+# it will be compiled with whatever latest processor the compiler supports!
+#
+CONFIG_RTE_MACHINE="default"
 
 CONFIG_RTE_ARCH="x86_64"
 CONFIG_RTE_ARCH_X86_64=y
