diff -up dpdk-2.0.0/lib/librte_pmd_null/rte_eth_null.c.orig dpdk-2.0.0/lib/librte_pmd_null/rte_eth_null.c
--- dpdk-2.0.0/lib/librte_pmd_null/rte_eth_null.c.orig	2015-04-07 15:54:49.437739442 -0400
+++ dpdk-2.0.0/lib/librte_pmd_null/rte_eth_null.c	2015-04-07 15:55:48.880447043 -0400
@@ -71,8 +71,13 @@ struct pmd_internals {
 	unsigned nb_rx_queues;
 	unsigned nb_tx_queues;
 
-	struct null_queue rx_null_queues[1];
-	struct null_queue tx_null_queues[1];
+	/*
+ 	 * NH these should both be 1, but we want to work
+ 	 * around a gcc compiler error here
+ 	 * as -Werror=array-bounds trips over this when its 1
+ 	 */
+	struct null_queue rx_null_queues[2];
+	struct null_queue tx_null_queues[2];
 };
 
 
